src/containers/ReviewsContainer.js

```diff
@@ -1,17 +1,14 @@
-import React from "react";
-import ReviewForm from "../components/ReviewForm";
-import Reviews from "../components/Reviews";
+import React from 'react';
+import ReviewForm from '../components/ReviewForm';
+import Reviews from '../components/Reviews';
 
-class ReviewsContainer extends React.Component {
-  render() {
-    return (
-      <div>
-        <ReviewForm toy={this.props.toy} />
-        <br />
-        <Reviews reviews={this.props.toy && this.props.toy.reviews} />
-      </div>
-    );
-  }
-}
+const ReviewsContainer = ({ toy }) => {
+  return (
+    <div>
+      <ReviewForm toy={toy} />
+      <Reviews reviews={toy && toy.reviews} />
```

Comment:
Not sure you were aware of this syntax, but optional chaining is your friend:

```suggestion
      <Reviews reviews={toy?.reviews} />
```

Reply:
Ohhh, that is fancy. Thank you!